gtda/images/tests/test_filtrations.py

```diff
@@ -68,8 +68,9 @@ def test_height_errors():
 
 @pytest.mark.parametrize("direction, images, expected",
                          [(None, images_2D, images_2D_height),
-                          ([1, 1], images_2D, images_2D_height),
-                          ([1, 0, 1], images_3D, images_3D_height)])
+                          (np.asarray([1, 1]), images_2D, images_2D_height),
+                          (np.asarray([1, 0, 1]), images_3D,
+                           images_3D_height)])
```

Comment:
Is  it  possible to parametrize the tests  also  over the type? (since everything throws errors anyway)

Reply:
I'm not sure I understand the question but I want to say "I don't see why not".